src/xcode/ENA/ENA/Source/HTTPClient/Resources/__tests__/SIDValidationDecoratorResourceTests.swift

```diff
@@ -0,0 +1,138 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class SIDValidationDecoratorResourceTests: XCTestCase {
+	
+	func test_If_RestService_Got_ClientError_Then_Abort() {
+		let expectation = expectation(description: "Expect that we got a completion")
+		
+		let stack = MockNetworkStack(
+			httpStatus: 404,
+			responseData: Data()
+		)
+		
+		let serviceProvider = RestServiceProvider(session: stack.urlSession)
+		guard let url = URL(string: "https://www.test.com") else {
+			XCTFail("Failed to create URL")
+			return
+		}
+		let resource = SIDValidationDecoratorResource(url: url)
+		serviceProvider.load(resource) { result in
+			switch result {
+			case .success:
+				XCTFail("client error is expected!")
+			case .failure(let error):
+				guard case let .receivedResourceError(resourceError) = error,
+					  .VD_ID_CLIENT_ERR == resourceError else {
+						  XCTFail("unexpected error case")
+						  return
+					  }
```

Comment:
```suggestion
				guard case let .receivedResourceError(. VD_ID_CLIENT_ERR) = error else {
						  XCTFail("Expected .VD_ID_CLIENT_ERR but received: \(error)")
						  return
					  }
```

Reply:
If you don't compare the error but just capture it (and for alle the remaining tests, too), you will not need to implement the Equatable protocol for `ServiceIdentityValidationDecoratorError`.

Reply:
yeah you are right, i had it first as i used to compare but now it is obsolete, i will remove the equatable conformance